src/lib/settings.ts

```diff
@@ -0,0 +1,18 @@
+import type { Settings } from "../client/nosto"
+import { nostojs } from "./nostojs"
+
+let settings: Settings | null = null
+
+if (window) {
```

Comment:
let's reuse the same pattern for window check:
if (typeof window !== "undefined") {